manylinux2010-x64/Toolchain.cmake

```diff
@@ -2,6 +2,8 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR x86_64)
 
+set(MANYLINUX2010 TRUE)
```

Comment:
Should we also set `set(MANYLINUX1 TRUE)`  ?

Reply:
I am also wondering if having the following would work better:

```
set(MANYLINUX TRUE)
set(MANYLINUX_VERSION 2010)
```

Reply:
Not sure. `MANYLINUX2010` doesn't mean `MANYLINUX1`. However, `MANYLINUX1` is compatible with `MANYLINUX2010`.

But then in your CMakeLists.txt, if you do `if (MANYLINUX1)` because of an old library there, maybe you don't want that to include `MANYLINUX2010`, that may have the newer library.

So I would say that having to use `if (MANYLINUX1 OR MANYLINUX2010)` would be less confusing.

Reply:
Any more thoughts on that regard? :slightly_smiling_face: